src/components/Modals/ModalProfile/styles.ts

```diff
@@ -0,0 +1,154 @@
+import styled, { css, keyframes } from 'styled-components'
+
+export const ModalProfile = styled.div`
+  ${() => css`
+    position: absolute;
+    right: 0;
+    top: 4.4rem;
+
+    z-index: 1057;
+
+    @media (max-width: 840px) {
+      width: 100vw;
+      height: 100vh;
+    }
+  `}
+`
+
+const OpenModal = keyframes`
```

Comment:
It's not a component, so I guess you should use camel case

```suggestion
const openModal = keyframes`
```